components/homme/src/preqx/share/prim_state_mod.F90

```diff
@@ -168,8 +168,7 @@ subroutine prim_printstate(elem, tl,hybrid,hvcoord,nets,nete)
     time1 = time - dt
 
 
-    ! npts = np
-    npts=SIZE(elem(1)%state%ps_v(:,:,n0),1)
+    npts=np
```

Comment:
**Not directly related to this PR, but general comment:**
Out of curiosity I dug into `global_integral` to see if this change has any impact there and I noticed that in the `global_integral` code `npts` is only passed to define the bounds on the input array for which we are integrating the global mass.  Within the body of the code the loop goes over `np`.  If we want the generality in `global_integral` to pass arbitrary sized arrays of `npts`X`npts` then why not have the loop indices in `global_integral` use `npts`?

Reply:
agreed - global_integral() used to support arbitrary size arrays, but now the arrays must be of size npts=np, and this argument should be removed.  Lets fix this in a separate PR.